Makefile.core.mk

```diff
@@ -360,12 +360,38 @@ gen: mod-download-go go-gen mirror-licenses format update-crds operator-proto co
 gen-check: gen check-clean-repo
 
 copy-templates:
+	# gateway charts
 	rm -r manifests/charts/gateways/istio-egress/templates
 	mkdir manifests/charts/gateways/istio-egress/templates
 	cp -r manifests/charts/gateways/istio-ingress/templates/* manifests/charts/gateways/istio-egress/templates
 	find ./manifests/charts/gateways/istio-egress/templates -type f -exec sed -i -e 's/ingress/egress/g' {} \;
 	find ./manifests/charts/gateways/istio-egress/templates -type f -exec sed -i -e 's/Ingress/Egress/g' {} \;
 
+	# external istiod remote cluster charts
+	cp manifests/charts/base/templates/reader-serviceaccount.yaml manifests/charts/istiod-remote/templates
+	cp manifests/charts/istio-control/istio-discovery/templates/mutatingwebhook.yaml manifests/charts/istiod-remote/templates
+	cp manifests/charts/istio-control/istio-discovery/templates/reader-clusterrole.yaml manifests/charts/istiod-remote/templates
+	cp manifests/charts/istio-control/istio-discovery/templates/reader-clusterrolebinding.yaml manifests/charts/istiod-remote/templates
+
+	# external istiod config cluster charts
+	cp manifests/charts/istio-control/istio-discovery/files/injection-template.yaml manifests/charts/istiod-remote/files
```

Comment:
nit: should we suffix all these with gen.yaml instead of .yaml? That will make them hidden on github and clear to developers not to modify

Reply:
I'm not sure if it's better for the files to have the exact same name as where they're copied from to make it clear they are the same files? Of course, it's probably clear enough if the only difference is the `.gen` part in the name.

I did add a [warning in NOTES.txt](https://github.com/istio/istio/pull/33527/files#diff-ec3c9922e17f1ff9cb76f6ec36365cd92f3ca0f1673dd671aaea3e2a45fc7e7c) not to edit them.